estolowka-frontend/src/app/auth/sessionUser.ts

```diff
@@ -0,0 +1,4 @@
+export interface SessionUser {
+    username: string;
```

Comment:
Change username for email due to fact that user will be login by email and password.